docs/releases/1.10.txt

```diff
@@ -214,6 +214,13 @@ Minor features
   ``{% load staticfiles %}`` or ``{% load static from staticfiles %}``) and
   not worry about whether or not the ``staticfiles`` app is installed.
 
+:mod:`django.utils.feedgenerator`
```

Comment:
Also, since this is a new feature, I think the release note should be documented in 1.11.txt.